main.go

```diff
@@ -78,7 +89,17 @@ func main() {
 	}()
 
 	server = core.NewNATSKafkaBridge()
-	server.InitializeFromFlags(flags)
+	err = server.InitializeFromFlags(flags)
```

Comment:
Printing/logging the error, stopping, and calling `os.Exit()` could be put in a function to reduce code duplication.